deployment/uberspace-ansible/files/generate_django_secret.j2

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ ! -f "{{ install_root }}.django-secret" ]; then
+    openssl rand -base64 32 > {{ install_root }}.django-secret
+fi
```

Comment:
Creating (and leaving) this file on the host to provision isn't very clean. Maybe use the ansible shell module instead.

Reply:
Oh yea I just found out about the "creates" option on the shell module :)